### ISSUE TITLE
Pending upgrade to RN@0.76

### ISSUE THREAD

**Issue by birdofpreyru** [birdofpreyru/react-native-static-server#123]

**Comment by birdofpreyru** [comment]
As of [v0.18.0](https://github.com/birdofpreyru/react-native-static-server/releases/tag/v0.18.0) the upgrade is done for **Android**, **iOS**, **macOS**.

The upgrade for **Windows** is pending the release of [react-native-windows](https://github.com/microsoft/react-native-windows) v0.76.

**Comment by hakansaglam29** [comment]
react-native-windows is now v0.76

**Comment by birdofpreyru** [comment]
It turns out, the upgrade to RN@0.76 for Windows is a large undertaking, which also requires to do the same upgrade in `react-native-fs` and `react-native-webview` libraries in parallel. I can't spent on it as much time as necessary, unless somebody sponsors it; in particular considering that I am not using RN for Windows at all myself. Thus, the ticket has been renamed and put _On Hold_.

**Comment by birdofpreyru** [comment]
Update: I haven't tested it yet, but presumably it should work just fine on Windows for legacy (UWP-based) apps.

**Comment by dppo** [comment]
> Update: I haven't tested it yet, but presumably it should work just fine on Windows for legacy (UWP-based) apps.

You are right, I just tested it, RN@0.76 for Legacy (based on UWP) can run
